ksqldb-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -354,22 +352,21 @@ private boolean isVariableSubstitutionEnabled() {
     return KsqlConfig.KSQL_VARIABLE_SUBSTITUTION_ENABLE_DEFAULT;
   }
 
-  private List<ParsedStatement> substituteVariables(final List<ParsedStatement> statements) {
+  private ParsedStatement substituteVariables(final ParsedStatement statement) {
     if (isVariableSubstitutionEnabled()) {
-      return statements.stream()
-          .map(stmt -> VariableSubstitutor.substitute(stmt, sessionVariables))
-          .flatMap(replacedSql -> KSQL_PARSER.parse(replacedSql).stream())
-          .collect(Collectors.toList());
+      final String replacedStmt = VariableSubstitutor.substitute(statement, sessionVariables);
+      return KSQL_PARSER.parse(replacedStmt).get(0);
```

Comment:
should we use `Iterables.getOnlyElement` here to make sure it fails if there's more than 1?